### ISSUE TITLE
Alter access to the host environment for flatpaks and maybe others in bauh? 

### ISSUE THREAD

**Issue by siduxdevil** [vinifmor/bauh#53]
Would it be possible to alter [access to the host environment](https://docs.flatpak.org/en/latest/sandbox-permissions.html) for flatpaks and maybe others in bauh?

**Comment by vinifmor** [comment]
Do you mean to apply the needed permissions through bauh ?

**Comment by siduxdevil** [comment]
Yes, grant or take away.

**Comment by vinifmor** [comment]
Sorry for the late reply. Do you mean the permissions to allow Flatpak apps installation ? ( the issue you had when installing Flatpaks through bauh )

**Comment by siduxdevil** [comment]
No, I mean the permissions for flatpaks to interact with the filesystem and other apps through portals etc.

**Comment by vinifmor** [comment]
Alright. I actually tried to port bauh as a Flatpak but couldn't get these permissions to work ( that's why it has no Flatpak release yet ). I will have a deeper look to see what's possible to do. Any help is welcome.

**Comment by siduxdevil** [comment]
Just found Flatseal [1]. Maybe the basics can be incorporated from there!?
[1] https://github.com/tchx84/flatseal

**Comment by vinifmor** [comment]
Looks promising. I'm going to test here. Thanks.
